### PR TITLE
Cart Page has been completed 

### PR DESCRIPTION
**Pull request by SarahSalem-hub**
### Note before Merge:

- There're  Array that have 2 objects for simulating the Cart data called "data", it will be removed once the local storage that set the data of the "adding to the cart"   be ready.
- Two libraries has been added : emotion, react-icons .
- I've been using emotion styling and code can be found in a single file in Cart's folder inside "Components".
### where files were added:

- **Cart folder** ---> **components**
- **cart.js** --->**page**: it's already exist. only updating the content.

### CODE REVIEW

**Comment by nabilashbat** [comment]
I'm noticing that you are trying to merge to the main branch and I guess that you didn't pull the changes that we've made on the develop branch, so what you need to do is you go back to your local repo, move the develop branch  and from the developer branch you push the changes to the develop branch you never go to the main.

**Comment by SarahSalem-hub** [comment]
**Thank you Nabil for pointing to this, yes I moved to develop and found the code, I created a branch from where I was like the screenshot below :** 

Screen Shot 2023-03-15 at 12 48 00 AM

**maybe I can fix this only by changing the "merge into" from the PR here :** 

Screen Shot 2023-03-15 at 12 28 06 PM

**Comment by nabilashbat** [comment]
If you do that from the pull request commits might be deleted, try to do it locally please, or read more and be more sure about 👍🏼

**Comment by nabilashbat** [comment]
It still didn't change in the GitHub pull requests panel
🤔

**Comment by SarahSalem-hub** [comment]
I didn't do anything yet

**Comment by SarahSalem-hub** [comment]
Hey @nabily4e-dev, I changed the base of the PR, and it worked fine 😊

File: components/cart/Cart.styled.jsx
Comment (nabilashbat) [inline components/cart/Cart.styled.jsx:1]: Well done 👏

**Review by nabilashbat — APPROVED** [review]

**Pull request merged**